RecoTracker/MkFit/plugins/MkFitEventOfHitsProducer.cc

```diff
@@ -63,6 +79,27 @@ void MkFitEventOfHitsProducer::produce(edm::StreamID iID, edm::Event& iEvent, co
   auto eventOfHits = std::make_unique<mkfit::EventOfHits>(mkFitGeom.trackerInfo());
   mkfit::StdSeq::Cmssw_LoadHits_Begin(*eventOfHits, {&pixelHits.hits(), &stripHits.hits()});
 
+  if (useStripStripQualityDB_) {
+    std::vector<mkfit::DeadVec> deadvectors(mkFitGeom.layerNumberConverter().nLayers());
+    const auto& siStripQuality = iSetup.getData(qualityToken_);
+    const auto& trackerGeom = iSetup.getData(geomToken_);
+    const auto& badStrips = siStripQuality.getBadComponentList();
+    for (const auto& bs : badStrips) {
+      const auto& surf = trackerGeom.idToDet(DetId(bs.detid))->surface();
+      const DetId detid(bs.detid);
+      bool isBarrel = (mkFitGeom.topology()->side(detid) == static_cast<unsigned>(TrackerDetSide::Barrel));
+      const auto ilay = mkFitGeom.mkFitLayerNumber(detid);
+      // dump content of deadmodules.h in standalone setup
+      // std::cout << "deadvectors["<<ilay<<"].push_back({"<<surf.phiSpan().first<<","<<surf.phiSpan().second<<","
+      // <<(isBarrel ? surf.zSpan().first : surf.rSpan().first)<<","<<(isBarrel ? surf.zSpan().second : surf.rSpan().second)<<"});"<<std::endl;
+      deadvectors[ilay].push_back({surf.phiSpan().first,
```

Comment:
as already discussed at the TRK POG meeting, this might be excessive since if I understand correctly even a single masked strip in a module results in the full module being masked.

Reply:
@mmusich, thanks for your comment.
The usage of the SiStrip quality DB information is disabled by default in this PR. At this point, we’re using this as  a starting point for exploring this further.